homeassistant/components/alexa/handlers.py

```diff
@@ -1523,3 +1524,30 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return directive.response()
+
+
+@HANDLERS.register(("Alexa.CameraStreamController", "InitializeCameraStreams"))
+async def async_api_initialize_camera_stream(hass, config, directive, context):
+    """Process a InitializeCameraStreams request."""
+    entity = directive.entity
+    camera_component = hass.data.get(camera.DOMAIN)
+    camera_entity = camera_component.get_entity(entity.entity_id)
+    stream_source = await camera.async_request_stream(
+        hass, camera_entity.entity_id, fmt="hls"
+    )
+    payload = {
+        "cameraStreams": [
+            {
+                "uri": f"{config.camera_stream_url}{stream_source}",
+                "protocol": "RTSP",
```

Comment:
This is weird. You request an HLS source but you tell Alexa it's RTSP ?

Reply:
Yes. :see_no_evil:

HLS is supported by all echo devices, but it's not documented for this API Capability. The echo device will still display the stream.